CMakeLists.txt

```diff
@@ -19,6 +19,22 @@ TRY YOUR LUCK
 #TODO Add pcre support
 #TODO exclude tests which are not supported
 
+#NOTE On Windows when compiling with MSVC or Clang you need add invalidcontinue.obj to the link options
+#This is VERY VERY IMPORTANT!!!
+
+#[[
+NOTE: when doing out of source builds the following tests fail.
+TODO make them relocatable
+t0200-gettext-basic.sh
+t0204-gettext-reencode-sanity.sh
+t7063-status-untracked-cache.sh
+t7800-difftool.sh
+t9902-completion.sh
+t9903-bash-prompt.sh
+t5310-pack-bitmaps.sh
+]]
```

Comment:
Can this be dropped? If not, I think we should fix the build until we can drop this comment.

Reply:
Will check these issues and get back to you about fixing them.